### DISCUSSION TITLE
[Feature] Album / Sub Album

### DISCUSSION THREAD

**Discussion by Luis-Lourenco** [immich-app/immich#4425] [Feature Request]
### The feature

The possibility of creating Albums over Album, or albums hierarchies. 

Also when import cli, recognise Album / Sub Album. 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2073

**Comment by Johnno987** [comment]
+1 Nested albums or lack thereof are whats holding me back from adopting Immich.
